tests/ftests/081-pybindings-cgrp_get_curr_ctrl_path-v1.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# cgroup_get_current_controller_path() test using the python bindings (cgroup v1)
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Kamalesh Babulal <kamalesh.babulal@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli, Mode
+from libcgroup import Cgroup, Version
+from process import Process
+import consts
+import ftests
+import sys
+import os
+
+
+CGNAME = '081getctrlpathv1'
+CONTROLLER = 'cpu'
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
+        return result, cause
+
+    if Cgroup.cgroup_mode() == Mode.CGROUP_MODE_UNIFIED:
+        result = consts.TEST_SKIPPED
+        cause = 'This test requires the legacy cgroup v1 hierarchy'
+
+    return result, cause
+
+
+def setup(config):
+    CgroupCli.create(config, CONTROLLER, CGNAME)
+
+    config.process.create_process_in_cgroup(config, CONTROLLER, CGNAME, ignore_systemd=True)
+
+
+def test(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    expected_path = os.path.join("/" + CGNAME)
+    pid = CgroupCli.get_pids_in_cgroup(config, CGNAME, CONTROLLER)[0]
+
+    cgrp = Cgroup(CGNAME, Version.CGROUP_V2)
+    #
+    # Test 1 - get the relative path of cgroup, for the pid's cpu controller.
+    #          It's expected to pass because we had created cgroup on cpu
+    #          hierarchy and moved the task to that group.
+    #
+    cgrp_path = cgrp.get_current_controller_path(pid, CONTROLLER)
+    if cgrp_path != expected_path:
+        result = consts.TEST_FAILED
+        cause = 'Expected cgroup path {} got {}'.format(expected_path, cgrp_path)
+
+    #
+    # Test 2 - get the relative path of cgroup, for the pid's memory controller.
+    #          It's expected to fail because we not had created cgroup.
+    #
+    cgrp_path = cgrp.get_current_controller_path(pid, "memory")
+    if cgrp_path == expected_path:
```

Comment:
```
# cgget -m
Legacy Mode (Cgroup v1 only).

# cat /proc/1054/cgroup 
13:pids:/user.slice/user-1000.slice/session-1.scope
12:blkio:/user.slice
11:net_cls,net_prio:/
10:memory:/user.slice/user-1000.slice/session-1.scope
9:perf_event:/
8:cpuset:/
7:misc:/
6:rdma:/
5:hugetlb:/
4:devices:/user.slice
3:cpu,cpuacct:/user.slice
2:freezer:/
1:name=systemd:/user.slice/user-1000.slice/session-1.scope
```

systemd might place it in another scope and not necessarily in '/', I too initially assumed it will be `root` cgroup, so limited the check `expected path`